dot/network/notifications.go

```diff
@@ -161,7 +160,6 @@ func (s *Service) createNotificationsMessageHandler(info *notificationsProtocol,
 				if err != nil {
 					logger.Trace("failed to validate handshake", "protocol", info.protocolID, "peer", peer, "error", err)
 					hsData.validated = false
-					_ = stream.Conn().Close()
```

Comment:
We no longer close connection stream when there is an error, why?  Is connection closing handled somewhere else?  In network/service.go line 500, the stream is closed when there is an error.  I'm just try to better understand how these streams are handled.

Reply:
yes, this was duplicate code, if this function returns an error then `readStream` in service.go will close the stream:
```
		err = handler(stream, msg)
		if err != nil {
			logger.Debug("failed to handle message from stream", "message", msg, "error", err)
			_ = stream.Close()
			return
		}
```